# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/openttd/hotfix-cmake.patch
# Copyright (C) 2024 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- openttd-13.4/CMakeLists.txt.vanilla	2024-03-14 19:13:18.304000128 +0100
+++ openttd-13.4/CMakeLists.txt	2024-03-14 19:13:42.208000140 +0100
@@ -8,10 +8,6 @@
     VERSION 13.4
 )
 
-if(CMAKE_SOURCE_DIR STREQUAL CMAKE_BINARY_DIR)
-    message(FATAL_ERROR "In-source builds not allowed. Please run \"cmake ..\" from the build directory. You may need to delete \"${CMAKE_SOURCE_DIR}/CMakeCache.txt\" first.")
-endif()
-
 # Debug mode by default.
 if(NOT CMAKE_BUILD_TYPE)
     set(CMAKE_BUILD_TYPE Debug)
